### ISSUE TITLE
About the validation set.

### ISSUE THREAD

**Issue by RSnewbie** [SDLZY/AD-DRL#2]
This work is very meaningful. I have a question about running the code: I found that when dividing the dataset, there are only training and testing sets. After saving the best parameters, the model still shows the results on the test set. However, the paper mentions extracting 10% of the parameters from the 80% training set for parameter tuning. How is the validation set implemented in this part? I look forward to your reply. Thank you.
